### PR TITLE
feature(semver-example): add example to better explain semver

### LINKED ISSUE #213: Improve SemVer introduction
**Is your feature request related to a problem? Please describe.**
I noticed SemVer description section could be more clearly described for those who are just learning about it.

**Describe the solution you'd like**
Adding an example img with some explanation.

### PR DESCRIPTION
**Pull request by nr431207**
## Changes
1. Added link to `SemVer` docs
2. Added img as a version example with explanation

## Purpose
To further clarify Semantic Versioning for those who are unfamiliar with this concept

## Approach
Included an image of a version with explanation for what each section of the version is indicating

Closes #213

### CODE REVIEW

File: standards/code-versioning.md [hunk 1]
```diff
@@ -2,83 +2,110 @@
...
+    -   Internal-facing code at a point that you or your team wants to demarcate as important
+    -   Versioning of your code (as a default, Shift3 follows SemVer)
+
+## [SemVer](https://semver.org/) Basics:
+
+-   Always have an up-to-date documented API (in our case, README.md for our projects or Wiki)
+-   Increment your versions based on a **Major.Minor.Patch** format where:
+
+    -   **Major** changes are breaking, non-backwards compatible changes to your code base.
+    -   **Minor** changes are new features or functionality that ius still backwards-compatable
```
Comment (michaelachrisco) [inline standards/code-versioning.md:16]: I believe `ius` should be `is`. That's a misspelling form the original material!

File: standards/code-versioning.md [hunk 2]
```diff
@@ -2,83 +2,110 @@
...
+
+    ##### Example:
+
+    ![example version](./assets/qa-process/semver.png)
+    In this instance:
+
+    -   We are on the `8`th major version, so there've been 7 changes that have broken backward compatibility in the past
+    -   We are on the `2`nd minor version which means there have been two feature changes that have been made
+    -   After we did our second feature push, we have had `6` bugs that we have fixed in the meantime.
+    -   If we build a new feature, this number will be changed to `8.3.0`.
```
Comment (michaelachrisco) [inline standards/code-versioning.md:27]: I would suggest being explicit on this feature example.

```suggestion
    -   If we build a new feature that does not break existing APIs, this number will be changed to `8.3.0`.
```

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Looks great, thanks for adding more documentation and an example image! Looks a lot better.

One misspelling and verbiage change if you dont mind.

File: standards/code-versioning.md (hunk 1 above)
Comment (nr431207) [reply standards/code-versioning.md:16]: great catch!

File: standards/code-versioning.md (hunk 2 above)
Comment (nr431207) [reply standards/code-versioning.md:27]: That's a great suggestion.

**Review by michaelachrisco — APPROVED** [review]
Wow your quick! Thanks for taking care of those items.

**Pull request merged**